sys/include/net/gnrc/udp.h

```diff
@@ -54,6 +54,22 @@ extern "C" {
 #endif
 
 /**
+ * @brief   Default UDP port range for dynamic source port allocation
+ */
+#define GNRC_UDP_DYN_PORTRANGE_MIN (49152U)
+#define GNRC_UDP_DYN_PORTRANGE_MAX (65535U)
+#define GNRC_UDP_DYN_PORTRANGE_NUM (GNRC_UDP_DYN_PORTRANGE_MAX - GNRC_UDP_DYN_PORTRANGE_MIN + 1)
+#define GNRC_UDP_DYN_PORTRANGE_ERR (0)
+
+/**
+ * @brief   Offset for next dynamic UDP port
+ *
+ * Currently set to a static (prime) offset, but could be random, too
+ * see https://tools.ietf.org/html/rfc6056#section-3.3.3
+ */
+#define GNRC_UDP_DYN_PORTRANGE_OFF (17U)
+
+/**
```

Comment:
I suggested it before, but how about the internal header `sys/net/gnrc/sock/include/gnrc_sock_internal.h`?